parts/kuberneteswindowssetup.ps1

```diff
@@ -177,11 +177,16 @@ Write-KubernetesStartFiles($podCIDR)
     $KubeletCommandLine = @"
 c:\k\kubelet.exe --hostname-override=`$global:AzureHostname --pod-infra-container-image=kubletwin/pause --resolv-conf="" --allow-privileged=true --enable-debugging-handlers --api-servers=https://`${global:MasterIP}:443 --cluster-dns=`$global:KubeDnsServiceIp --cluster-domain=cluster.local  --kubeconfig=c:\k\config --hairpin-mode=promiscuous-bridge --v=2 --azure-container-registry-config=c:\k\azure.json --runtime-request-timeout=10m
 "@
-
+    # stop using container runtime interface from 1.6.0+ (officially deprecated from 1.7.0)
+    if ($global:KubeBinariesVersion -ge "1.6.0")

```

Comment:
Fair enough. In that case, I think you could group both checks under one check for v1.6.0 or greater.